geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -41,10 +44,16 @@ object HBaseFeatureIndex extends HBaseIndexManagerType {
 
   val DataColumnQualifier = Bytes.toBytes("d")
   val DataColumnQualifierDescriptor = new HColumnDescriptor(DataColumnQualifier)
+
+  case class ScanConfig(hbaseFilters: Seq[HBaseFilter],
+                        columnFamily: Array[Byte],
+                        entriesToFeatures: Iterator[Result] => Iterator[SimpleFeature],
+                        reduce: Option[(CloseableIterator[SimpleFeature]) => CloseableIterator[SimpleFeature]])
```

Comment:
reduce is also currently incorporated into `entriesToFeatures`, and isn't used, so we should remove it for now